@W-23713942-Document API Manager platform request limits (#669)
mulesoft/docs-api-manager: modules/ROOT/pages/limits.adoc

| 
|===

[[api-manager-platform-request-limits]]

> == API Manager Platform Request Limits

Reviewer: ```suggestion
== Control Plane API Rate Limits

> Anypoint Platform enforces rate limits on requests to certain API Manager control plane endpoints to protect service stability and ensure fair use. These limits apply to calls made to the API Manager backend APIs, not to the API instances that you manage.

Reviewer: ```suggestion
Anypoint Platform applies rate limits per client IP address to requests sent to the following API Manager control plane endpoints. These limits don't apply to requests sent to your managed API instances.

> When a request exceeds the rate limit for an endpoint, the platform returns a `429` status code.

Reviewer: ```suggestion
When requests from a client IP address exceed an endpoint's rate limit, Anypoint Platform returns a 429 (Too Many Requests) status code.
```

> [%header%autowidth.spread,cols="a,a,a"]

Reviewer: ```suggestion
[%header%autowidth.spread,cols="a,>.<a,a"]
```

|===

> |Endpoint |Rate Limit Per Client IP Address | Status Code

Reviewer: ```suggestion
|Endpoint |Rate Limit | Status Code
```
